src/serverSetup/nunjucks.js

```diff
@@ -16,6 +16,9 @@ const proto = {
     logger.info(`rendering: ${name} with schema=<${schema ? 'defined' : 'any'}>`)
     return customRenderer.render(nunjucks, name, schema ?? v.any(), context)
   },
+  renderString (string, context) {
+    return nunjucks.renderString(string, context)
+  },
```

Comment:
added this as used for testing table components and potentially other components. without needing a template file